doc/pgtap.mmd

```diff
@@ -7960,6 +7986,93 @@ missing policy command, like so:
     #         have: INSERT
     #         want: ALL
 
+### `rls_is_enabled()` ###
+
+    SELECT rls_is_enabled( :schema, :table, :desired_value );
+
+Test whether [row-level security](https://www.postgresql.org/docs/current/ddl-rowsecurity.html)
+is enabled (`:desired_value` true) or disabled (`:desired_value` false).
```

Comment:
The description should go _after_ the parameters to be consistent with the existing docs. Same goes for some of the other documentation added here.